### ISSUE TITLE
OTP auth doesn't work with Bitwarden import

### ISSUE THREAD

**Issue by zoktiik** [roddhjav/pass-import#203]
`pass otp` seems to require that all TOTP secrets are wrapped in an `otpauth://` URI. Upon importing, my Bitwarden vault, the TOTP secrets are just dumped in the file at the end, without actually being converted to the proper format. As such, attempting to use `pass otp` with a `pass` entry that has one doesn't work, but it does work when I manually wrap the secret in that URI format.
My initial import was using `-c -C -a`, but I also did an import without specifying options and got a similar result. I used JSON format.

**Comment by roddhjav** [comment]
Can ensure the format of your json file is the same than this one: https://github.com/roddhjav/pass-import/blob/master/tests/assets/db/bitwarden.json

**Comment by zoktiik** [comment]
Yes, it is in the same format.
Example output from import:
```
[password]
login: [username]
url: https://example.com/
comments: Lorem ipsum dolor sit amet.
[raw TOTP secret]
url2: androidapp://com.acme.example
```

**Comment by roddhjav** [comment]
This should be fixed by [e4823ee](https://github.com/roddhjav/pass-import/commit/e4823eec509c35f151af69b9f339e9df2f07904e)
